api/v1alpha1/profilecatalogsource_types.go

```diff
@@ -58,7 +58,7 @@ type ProfileCatalogEntry struct {
 	Tag string `json:"tag,omitempty"`
 	// CatalogSource is the name of the catalog the profile is listed in
 	// +optional
-	CatalogSource string `json:"catalog,omitempty"`
+	CatalogSource string `json:"catalogSource,omitempty"`
```

Comment:
breaking changes :ahhhhh:

Reply:
The breaking change was to not update this when the variable was renamed hahahahaha

Reply:
🙉

Reply:
:upside_down_face: